.github/workflows/elixir-sdk.yml

```diff
@@ -0,0 +1,66 @@
+name: 'Elixir SDK code checks'
+
+on:
+    push:
+        branches: ['main']
+    pull_request:
+        types: [opened, reopened, synchronize, assigned, review_requested]
+        paths:
+            - 'sdk/highlight-ex/**'
+            - '.github/workflows/elixir-sdk.yml'
+
+env:
+    ELIXIR_VERSION: "1.13.4"
+    OTP_VERSION: "25.0.1"
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+jobs:
+    format:
+        name: format elixir files
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+            - name: Setup Elixir
+              uses: actions/setup-elixir@v1
+              with:
+                elixir-version: ${{ env.ELIXIR_VERSION }}
+                otp-version: ${{ env.OTP_VERSION }}
+            - name: Install Dependencies
+              run: mix format
+
+    test:
+        name: test package
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+            - name: Setup Elixir
+              uses: actions/setup-elixir@v1
+              with:
+                elixir-version: ${{ env.ELIXIR_VERSION }}
+                otp-version: ${{ env.OTP_VERSION }}
+            - name: Install Dependencies
+              run: mix deps.get
+            - name: Run Tests
+              run: mix test
+
+    publish:
+        name: publish package
+        runs-on: ubuntu-latest
+        if: github.ref == 'ref/heads/main'
+        needs: test
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+            - name: Setup Elixir
+              uses: actions/setup-elixir@v1
+              with:
+                elixir-version: ${{ env.ELIXIR_VERSION }}
+                otp-version: ${{ env.OTP_VERSION }}
+            - name: Publish to Hex
+              uses: synchronal/hex-publish-action@v3
+              with:
+                name: highlight
+                key: ${{ secrets.HEX_PM_KEY }}
```

Comment:
how do i need to set this up? do i need to create an account somewhere?

Reply:
https://hex.pm/docs/publish